lib/flutter_calendar_carousel.dart

```diff
@@ -106,6 +106,8 @@ class CalendarCarousel extends StatefulWidget {
   final DateTime maxSelectedDate;
   final TextStyle inactiveDaysTextStyle;
   final TextStyle inactiveWeekendTextStyle;
+  final bool headerTitleTouchable;
```

Comment:
Do we actually need this variable? I think `onHeaderTitlePressed` would be enough because you can distinguish whether this is `null`.

Reply:
I also thought about that but maybe someone wants to save it's current behavior. If you approve `headerTitleTouchable` deletion, I will do PR without it.

Reply:
Sounds reasonable. I'll resolve this conversation.